.env.example

```diff
@@ -13,7 +13,6 @@ VALIDATION_TEMPLATE="validation-template.xlsx"
 # Agency input spreadsheet archives
 # This is /var/data/uploads in the states' deployed instances
 UPLOAD_DIRECTORY="./uploads"
-DATA_DIR="./datadir"
```

Comment:
my goal was to have a single env var, which is `DATA_DIR`, vs lots of sub-env-vars like `UPLOAD_DIR` or `REPORT_DIR`